typegate/src/runtimes/deno/deno_messenger.ts

```diff
@@ -31,8 +31,8 @@ export class DenoMessenger extends LazyAsyncMessenger<Worker, Task, unknown> {
               ...permissions,
               // non-overridable permissions (security between typegraphs)
               run: false,
-              read: false,
-              write: false,
+              read: "inherit",
```

Comment:
give explicit read access to the tmp folder, otherwise typegraph can read other typegraph